src/Akeneo/Bundle/ElasticsearchBundle/Client.php

```diff
@@ -66,6 +66,34 @@ public function index($indexType, $id, array $body)
 
     /**
      * @param string $indexType
+     * @param array  $bodies
+     *
+     * @return array see {@link https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_indexing_documents.html#_bulk_indexing}
+     */
+    public function bulkIndexes($indexType, $bodies)
+    {
+        $params = [];
+
+        foreach ($bodies as $body) {
+            $index = [
+                '_index' => $this->indexName,
+                '_type'  => $indexType
+            ];
+
+            if (isset($body['_id'])) {
```

Comment:
should we thrown an exception if `_id` is missing ? because if missing, we will have a lot a duplicated products